metrics_collector/manager_collector.go

```diff
@@ -12,10 +12,12 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/util"
 )
 
 type ManagerCollector struct {
 	*baseCollector
+	proxyConnCounter util.Counter
```

Comment:
Does this collector require the counter?

Reply:
Removed.